tests/test_s3api_policy_object.sh

```diff
@@ -286,67 +310,76 @@ test_s3api_policy_put_wildcard() {
   policy_file="policy_file"
   test_folder="test_folder"
   test_file="test_file"
-  username=$USERNAME_ONE
-  password=$PASSWORD_ONE
 
   run create_test_folder "$test_folder"
   assert_success
 
   run create_test_files "$test_folder/$test_file" "$policy_file"
   assert_success
 
+  run setup_user_versitygw_or_direct "$USERNAME_ONE" "$PASSWORD_ONE" "user" "$BUCKET_ONE_NAME"
+  assert_success
+  # shellcheck disable=SC2154
+  username=${lines[0]}
```

Comment:
Is this assumed the setup_user_versitygw_or_direct returns a variable called "lines"?  This could be a direct return from the function and would make the code more readable.

Reply:
"lines" is a variable returned by bats (output lines in an array).  Is there another way to do direct returns in bats?